icu_benchmarks/run_utils.py

```
@@ -15,6 +15,9 @@
 from statistics import mean, pstdev
 from icu_benchmarks.models.utils import JsonResultLoggingEncoder
 from icu_benchmarks.wandb_utils import wandb_log
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)